SingleSource/UnitTests/2024-indirect-branch.c

```diff
@@ -0,0 +1,18 @@
+__attribute__((noinline))
+unsigned add(unsigned a, unsigned b) {
+    return a + b;
+}
+
+int main() {
+    // Declare a function pointer and initialize it to point to the add function
+    unsigned (*funcPtr)(unsigned, unsigned) = add;
+    // Now, call the add function indirectly using the function pointer
+    unsigned result = funcPtr(5, 3);
+
+    if (result != 8) {
```

Comment:
You can use our builtin to write to stdout now. `__builtin_delendum_write((int)x)`

Reply:
great! how in the test framework to check stdout prints?

Reply:
The stdout goes to the `.log` file. So you can check there. There isn't a better test framework support. Maybe we should add that to the script. One can compare the expected result with the content in the `.log` file. If you want to do it the old way, could you add the positive test as well like the other tests? (So add -neg` to this file's name, and add another one that tests equality) Thanks!

Reply:
Guess there is a new work item -- revamp the testing script and use .log file for expected value checking.

I think the old way should be ditched right away -- checking log file is way better. We should discuss about and agree on a test framework for it.